edge/pkg/metamanager/metaserver/kubernetes/scope/scope.go

```diff
@@ -46,26 +48,32 @@ func NewRequestScope() *handlers.RequestScope {
 
 		Serializer:     serializer.NewNegotiatedSerializer(),
 		ParameterCodec: scheme.ParameterCodec,
-		//Creater:         nil,
-		Convertor: fakers.NewFakeObjectConvertor(),
-		Defaulter: fakers.NewFakeObjectDefaulter(),
-		Typer:     crdserverscheme.NewUnstructuredObjectTyper(),
-		//UnsafeConvertor: nil,
-		Authorizer: fakers.NewAlwaysAllowAuthorizer(),
+
+		StandardSerializers: make([]runtime.SerializerInfo, 0),
+
+		Creater:         unstructuredscheme.NewUnstructuredCreator(),
+		Convertor:       fakers.NewFakeObjectConvertor(),
+		Defaulter:       fakers.NewFakeObjectDefaulter(),
+		Typer:           crdserverscheme.NewUnstructuredObjectTyper(),
+		UnsafeConvertor: fakers.NewFakeObjectConvertor(),
+		Authorizer:      fakers.NewAlwaysAllowAuthorizer(),
 
 		EquivalentResourceMapper: runtime.NewEquivalentResourceRegistry(),
 
-		TableConvertor: nil,
-		FieldManager:   fakeFieldManager,
+		TableConvertor: rest.NewDefaultTableConvertor(schema.GroupResource{}),
```

Comment:
Is this also required to change against the panic? could you share the explanation why this is needed?

Reply:
`TableConvertor` is used in `kubeedge/vendor/k8s.io/apiserver/pkg/endpoints/handlers/response.go`, if it is called while it is `nil`, another panic will happen. I just want to prevent this in advance.